tests/model_registry/test_rest_api_stateful.py

```diff
@@ -8,10 +8,9 @@
 
 @pytest.mark.fuzzer
 @pytest.mark.parametrize(
-    "model_registry_namespace, updated_dsc_component_state_scope_class",
+    "updated_dsc_component_state_scope_class, model_registry_namespace",
```

Comment:
Applicable to the other files as well, but why do you still parametrize `model_registry_namespace`?
The changes to tests/model_registry/conftest.py#L50 make it return from `updated_dsc_component_state_scope_class` directly

Reply:
Missed this one. I will update it.